pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -437,12 +437,12 @@ class NativeAssetsBuildRunner {
         ),
       );
       if (buildOutput == null) return null;
-      hookResult = hookResult.copyAdd(buildOutput);
+      hookResult = hookResult.copyAdd(buildOutput, [/*dry run is not cached*/]);
```

Comment:
Didn't you say dry run was finally purged from existence?
Why is this code still here?